lib/Rex/Commands/Augeas.pm

```diff
@@ -369,6 +369,25 @@ Returns the value of the given item.
     }
   }
 
+=item errors
+
+Returns any errors from a previous command (uses the augeas "errors" command)
+
+ my $errors = augeas "errors"
```

Comment:
What's the format returned? Is a scalar a good fit, or it would be better as an array (as indicated by the plural in `errors`)?

Also perhaps add a `;` at the end.